docs/stencil-docs/getting-started/transitioning-to-stencil.md

```diff
@@ -5,149 +5,88 @@
 ### On This Page
 - [Transitioning to Stencil](#Transitioning-to-Stencil)
     - [On This Page](#On-This-Page)
-  - [Find an Agency Partner](#Find-an-Agency-Partner)
-  - [Adapt a Stencil Theme or Develop from Scratch](#Adapt-a-Stencil-Theme-or-Develop-from-Scratch)
-  - [Transition to Stencil from Blueprint](#Transition-to-Stencil-from-Blueprint)
-    - [Technical Differences](#Technical-Differences)
-      - [PHP vs Handlebars](#PHP-vs-Handlebars)
-    - [WebDav vs CLI](#WebDav-vs-CLI)
-    - [Customizing checkout with Stencil](#Customizing-checkout-with-Stencil)
-    - [Updating a theme](#Updating-a-theme)
-    - [FAQ](#FAQ)
-  - [SEO Tips and Best Practices](#SEO-Tips-and-Best-Practices)
-  - [Interacting with the BigCommerce API](#Interacting-with-the-BigCommerce-API)
-    - [Partnering with BigCommerce](#Partnering-with-BigCommerce)
+  - [What is Stencil?](#What-is-Stencil?)
+  - [Why Transition to Stencil?](#Why-Transition-to-Stencil?)

```

Comment:
I personally would still rather the heading be "Transitioning to Stencil" instead of Why Transition to Stencil?, but I think that's my personal preference. :)